client/src/features/ProjectPageV2/ProjectPageContent/Settings/ProjectSettingsMembers.tsx

```diff
@@ -165,11 +168,17 @@ function ProjectPageSettingsMembersListItem({
   onRemove,
   onEdit,
 }: ProjectPageSettingsMembersListItemProps) {
+  const nameToDisplay = getMemberNameToDisplay(member);
   return (
     <ListGroupItem>
       <Row className="g-2">
         <Col className="align-content-around" xs={12} md="auto">
-          {member.email ?? member.id}{" "}
+          {nameToDisplay}{" "}
+          {member.namespace ? (
+            <span className="fst-italic">@{member.namespace} </span>
+          ) : (
+            ""
+          )}
```

Comment:
Building on comment in `roleUtils`:

```suggestion
          <ProjectMemberDisplay member={member} />
```